### PR TITLE
Add a DaemonSet to set the login grace time for ssh server for GDCV clusters

### PR DESCRIPTION
**Pull request by Marinette101**
GDCV clusters require `tolerations` and `resources.requests` settings additionally.

### CODE REVIEW

File: ssh-server-config/README.md [hunk 1]
```diff
@@ -6,12 +6,23 @@ The ssh-server-config tool is a Kubernates DaemonSet that set [loginGraceTime](h
...
+
+### GKE Clusters
+
 ```
 kubectl apply -f \
 https://raw.githubusercontent.com/GoogleCloudPlatform\
 /k8s-node-tools/master/ssh-server-config/set-login-grace-time.yaml
 ```
 
+### GDCV Clusters
```
Comment (Random-Liu) [inline ssh-server-config/README.md]: GDC software-only for VMware

File: ssh-server-config/set-login-grace-time-gdcv.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    matchLabels:
+      app: ssh-server-config
+  template:
+    metadata:
+      labels:
+        app: ssh-server-config
+    spec:
+      hostPID: true
+      tolerations:
+        - key: node-role.kubernetes.io/control-plane
```
Comment (Random-Liu) [inline ssh-server-config/set-login-grace-time-gdcv.yaml]: The security incident exists in older version of clusters, so we may want to tolerate the old master taint as well.

Actually, we may just tolerate all taints.

**Review by vinayakankugoyal — APPROVED** [review]

**Comment by vinayakankugoyal** [comment]
@bobbypage could you merge this for us.

**Comment by yguo0905** [comment]
Can you please wait for the comments to be resolved and then submit?

File: ssh-server-config/set-login-grace-time-gdcv.yaml (hunk 2 above)
Comment (vinayakankugoyal) [reply ssh-server-config/set-login-grace-time-gdcv.yaml]: +1

File: ssh-server-config/README.md (hunk 1 above)
Comment (Marinette101) [reply ssh-server-config/README.md]: Done

File: ssh-server-config/set-login-grace-time-gdcv.yaml (hunk 2 above)
Comment (Marinette101) [reply ssh-server-config/set-login-grace-time-gdcv.yaml]: Changed to tolerate all taints

**Comment by bobbypage** [comment]
LGTM

**Pull request merged**